app/assets/stylesheets/collections/bulk-edit.scss

```diff
@@ -0,0 +1,51 @@
+.bulk-edit{
+  &-container {
+    display:flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: stretch;
+    height: calc(100vh - #{$nav-height} - #{$footer-height}); 
+    min-height: 200px;
+    @media only screen and (max-width: 850px) {
+      width: 90%;
+      height: calc(100vh - #{$nav-height} - #{$footer-height} - 15px); 
+    }
+  }
+
+  &_error,
+  &_form{
+    flex: 1 0 auto;
```

Comment:
```suggestion
    flex: 1 0 auto;
    min-height: 300px;
```

Reply:
This should've been in my review. Just suggesting to add a min height to the textarea, since it defaults to 2 lines tall on my phone.

Reply:
Good catch, I'll put this down on the &_form selector down on line 30.

Reply:
fixed